lib/logger.js

```diff
@@ -17,6 +17,16 @@ var request = require('request');
 var graylog2 = require('graylog2');
 var Graylog = graylog2.graylog;
 
+sentry = {
+  dsn: isProd ? process.env.SENTRY_DNS : undefined,
+  integrations: [new Dedupe()], // dedupe based on fingerprint and stack trace
+  tracesSampleRate: 1, // 100% of all errors will be reported
+  environment: isProd ? 'production' : 'testing',
+  serverName: require('os').hostname()
+}
+const Sentry = require('@sentry/node')
+Sentry.init({ dsn: sentry.dsn })
+
```

Comment:
I mean, with this code you must be 1000% sure that you use same settings everywhere

Reply:
that's just a draft... of course the must be removed. the config belongs the the repos using it.
it's just about protecting the code from sentry